app/controllers/admin/images_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Admin::ImagesController < Admin::BaseController
+  before_action :find_edition
+  before_action :enforce_permissions!
+  before_action :build_blank_image
+  layout "design_system"
+
+  def index; end
+
+private
+
+  def find_edition
+    edition = Edition.find(params[:edition_id])
+    @edition = LocalisedModel.new(edition, edition.primary_locale)
+  end
+
+  def build_blank_image
+    if @edition.allows_image_attachments? && @edition.images.none?(&:new_record?)
+      image = @edition.images.build
+      image.build_image_data
+    end
+  end
+
+  def enforce_permissions!
```

Comment:
i think we need to enforce a similar permission to the edition summary page. This is from the editions controller

```
  def enforce_permissions!
    case action_name
    when "index", "topics", "history"
      enforce_permission!(:see, edition_class || Edition)
    when "show"
      enforce_permission!(:see, @edition)
    when "new"
      enforce_permission!(:create, edition_class || Edition)
    when "create"
      enforce_permission!(:create, @edition)
    when "edit", "update", "revise", "diff", "update_bypass_id"
      enforce_permission!(:update, @edition)
    when "destroy", "confirm_destroy"
      enforce_permission!(:delete, @edition)
    when "export", "confirm_export"
      enforce_permission!(:export, edition_class || Edition)
    else
      raise Whitehall::Authority::Errors::InvalidAction, action_name
    end
  end
```

So i think we just need 

```
enforce_permission!(:see,  Edition)
```

then for the new/edit/create/update actions later it would be 

```
:create, Edition
```